src/main/java/de/tum/in/www1/artemis/service/connectors/GitService.java

```diff
@@ -1247,7 +1247,7 @@ public Path zipRepositoryWithParticipation(Repository repo, String repositoryDir
             zipRepoName += "-student-submission.git.zip";
         }
         else {
-            zipRepoName += "-" + studentTeamOrDefault + ".zip";
+            zipRepoName += "-" + studentTeamOrDefault + (participation.isTestRun() ? "_practice" : "") + ".zip";
```

Comment:
We already have addPracticePrefixIfTestRun() in Participation. Maybe you can reuse it?

Reply:
Do you think it makes sense that it is a prefix instead of a suffix? 
<img width="444" alt="image" src="https://user-images.githubusercontent.com/5898705/225293608-7cde2025-4ea3-4f75-a2b4-da5edb9cb47e.png">

i.e. `P101-Programming_Exercise_1-4-practice-artemis_test_user_1` using `addPracticePrefixIfTestRun()`

Reply:
Ok, your version looks nicer. Maybe you can create a similar method and extract "practice" to a common constant?